.github/workflows/release.yaml

```diff
@@ -78,18 +78,19 @@ jobs:
               autonomy init --reset --author $AUTHOR --ipfs --remote
               autonomy fetch $AUTHOR/$SERVICE --service --local || exit 1
               cd $SERVICE || exit 1
-              autonomy build-image || exit 1
-              autonomy build-image --version $VERSION || exit 1
-
+              autonomy build-image --image-author $DOCKER_USER || exit 1
+              docker tag $DOCKER_USER/oar-$SERVICE:$DEFAULT_IMAGE_TAG $DOCKER_USER/oar-$SERVICE:$VERSION || exit 1
+              docker tag $DOCKER_USER/oar-$SERVICE:$DEFAULT_IMAGE_TAG $DOCKER_USER/oar-$SERVICE:latest || exit 1
```

Comment:
Is this necessary?
Also the image we are building is the agent image. In this case it shouldnt be a problem because the service name matches the agent name.

Reply:
> Is this necessary?

Do you mean tagging with `latest`? No, it is not necessary just preferrable.

> Also the image we are building is the agent image. In this case it shouldnt be a problem because the service name matches the agent name.

Yes, this is correct. I just followed the same approach as previously. For example, if you check the deleted code below, you will notice that the service was being used. e.g.:

```
echo "Pushing $DOCKER_USER/$SERVICE:$VERSION"
```

We could add a:

```
grep 'agent' packages/packages.json | awk -F/ '{print $3}' | head -1
```

for the agent. What do you think?